app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,161 @@ function Page() {
+                this way, and to show the outline here in the styleguide.</p>
+              <div><pre><code>{`<ContentContainer style={{'border': '1px solid var(--purple-600)'}}>
+...
+</ContentContainer>`}
+              </code></pre></div>
+            </ContentContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-content-padding</code></dt>
+              <dd>
+                Set the content container padding, defaults to: <code>var(--brand-space2)</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>ToolbarContainer</code> (LayoutContainer)</h2>
+            <p><code>LayoutContainer > ToolbarContainer</code></p>
+            <ToolbarContainer style={{'--layout-container-background': 'var(--slate-200)'}}>
+              <p>Some toolbar text, maybe a Title</p>
+              <Button>Button</Button>
+            </ToolbarContainer>
+
+            <h3>CSS Variables</h3>
+            <p>Since this inherrits from <code>LayoutContainer</code> those CSS
+              variables also works. In addition to those, the following is also available:</p>
+            <dl>
+              <dt><code>--layout-gap</code></dt>
+              <dd>
+                This container uses flexbox, and this variable sets
+                the <code>gap</code> css property. Defaults to <code>1em</code>
+              </dd>
+            </dl>
+
+            <h2>Sidebar Container (Composite)</h2>
+            <p>This is a composite layout component than contains a sidebar
+              alongside a content container.</p>
+            <p><code>LayoutContainer > LayoutWithSidebar</code></p>
+            <p><code>ContentContainer > SidebarContainer</code></p>
+            <ToolbarContainer>
+              <Button onPress={toggleSidebarDirection}>Toggle Direction</Button>
+              <Button onPress={toggleSidebar}>Toggle Sidebar</Button>
+            </ToolbarContainer>
+            <LayoutWithSidebar sidebarPosition={sidebarDirection} className="sg-sidebar">
+              <SidebarContainer isOpen={sidebarOpen}>
+                <p>This is the sidebar for the component</p>
+              </SidebarContainer>
+
+              <ContentContainer>
+                <h3>Layout With Sidebar</h3>
+                <p>This is the primary content for the slider component</p>
+
+                <div><pre><code>
+                  {`<LayoutWithSidebar direction="right">
+  <SidebarContainer> ... </SidebarContainer>
+  <ContentContainer> ... </ContentContainer>
+</LayoutWithSidebar>
+`}
+                </code></pre></div>
+              </ContentContainer>
+            </LayoutWithSidebar>
+
+            <p><code>LayoutWithSidebar</code> can have a
+              <code>ToolbarContainer</code> in addition to the sidebar.</p>
```

Comment:
L1175: please add a space in front of "ToolbarContainer" and one after the word "outside on line 1177.